### PR TITLE
Run dotnet-format

### PR DESCRIPTION
**Pull request by bricelam**
These are essentially the places where Visual Studio code cleanup disagrees with the current formatting.

We might be able to tweak `.editorconfig` a little more to minimize the differences...

Could also try running with `--fix-style` just to see what happens

### CODE REVIEW

File: src/EFCore.Abstractions/Infrastructure/ILazyLoader.cs [hunk 1]
```diff
@@ -32,15 +32,15 @@ public interface ILazyLoader : IDisposable
         /// <param name="loaded"> Determines whether the navigation is set as loaded or not. </param>
         void SetLoaded(
             [NotNull] object entity,
-            [NotNull] [CallerMemberName] string navigationName = "",
+            [NotNull][CallerMemberName] string navigationName = "",
```
Comment (bricelam) [inline src/EFCore.Abstractions/Infrastructure/ILazyLoader.cs:35]: 👎

File: src/EFCore/Infrastructure/EntityFrameworkEventSource.cs [hunk 2]
```diff
@@ -111,7 +111,8 @@ protected override void OnEventCommand(EventCommandEventArgs command)
                 _queriesPerSecondCounter ??= new IncrementingPollingCounter(
                     "queries-per-second",
                     this,
-                    () => _totalQueries) { DisplayName = "Queries", DisplayRateTimeScale = TimeSpan.FromSeconds(1) };
+                    () => _totalQueries)
+                { DisplayName = "Queries", DisplayRateTimeScale = TimeSpan.FromSeconds(1) };
```
Comment (bricelam) [inline src/EFCore/Infrastructure/EntityFrameworkEventSource.cs:115]: 👎

File: src/EFCore/Metadata/Internal/Index.cs [hunk 3]
```diff
@@ -101,7 +101,8 @@ public class Index : ConventionAnnotatable, IMutableIndex, IConventionIndex
         /// </summary>
         public virtual InternalIndexBuilder Builder
         {
-            [DebuggerStepThrough] get;
+            [DebuggerStepThrough]
+            get;
             [DebuggerStepThrough]
```
Comment (bricelam) [inline src/EFCore/Metadata/Internal/Index.cs:106]: I'd prefer another newline between these now.

File: src/EFCore.Cosmos/Query/Internal/SqlExpressionFactory.cs [hunk 4]
```diff
@@ -175,7 +175,7 @@ public virtual SqlExpression ApplyTypeMapping(SqlExpression sqlExpression, CoreT
                     resultType = typeof(bool);
                     resultTypeMapping = _boolTypeMapping;
                 }
-                    break;
+                break;
```
Comment (smitpatel) [inline src/EFCore.Cosmos/Query/Internal/SqlExpressionFactory.cs:178]: Should put this break inside the block.

File: test/EFCore.Cosmos.Tests/CosmosApiConsistencyTest.cs [hunk 5]
```diff
@@ -45,7 +45,8 @@ public override bool TryGetProviderOptionsDelegate(out Action<DbContextOptionsBu
 
             public override
                 List<(Type Type, Type ReadonlyExtensions, Type MutableExtensions, Type ConventionExtensions, Type
-                    ConventionBuilderExtensions)> MetadataExtensionTypes { get; }
+                    ConventionBuilderExtensions)> MetadataExtensionTypes
+            { get; }
```
Comment (bricelam) [inline test/EFCore.Cosmos.Tests/CosmosApiConsistencyTest.cs:49]: 👎

File: test/EFCore.Design.Tests/Migrations/Design/CSharpMigrationOperationGeneratorTest.cs [hunk 6]
```diff
@@ -2535,7 +2535,8 @@ public void SqlOperation_required_args()
         };
 
         private static readonly MultiPoint _multiPoint = new MultiPoint(
-            new[] { new Point(1.1, 2.2), new Point(2.2, 2.2), new Point(2.2, 1.1) }) { SRID = 4326 };
+            new[] { new Point(1.1, 2.2), new Point(2.2, 2.2), new Point(2.2, 1.1) })
+        { SRID = 4326 };
```
Comment (bricelam) [inline test/EFCore.Design.Tests/Migrations/Design/CSharpMigrationOperationGeneratorTest.cs:2539]: 👎

File: test/EFCore.Specification.Tests/Query/GearsOfWarQueryTestBase.cs [hunk 7]
```diff
@@ -3279,7 +3281,7 @@ public virtual Task Comparing_two_collection_navigations_composite_key(bool asyn
                 async,
                 ss => from g1 in ss.Set<Gear>()
                       from g2 in ss.Set<Gear>()
-                      // ReSharper disable once PossibleUnintendedReferenceComparison
+                          // ReSharper disable once PossibleUnintendedReferenceComparison
```
Comment (bricelam) [inline test/EFCore.Specification.Tests/Query/GearsOfWarQueryTestBase.cs:3284]: 👎

File: src/EFCore/Metadata/Internal/Index.cs (hunk 3 above)
Comment (AndriySvyryd) [reply src/EFCore/Metadata/Internal/Index.cs:106]: I'd prefer them to remain on the same line for the simple cases

Comment (bricelam) [reply src/EFCore/Metadata/Internal/Index.cs:106]: K. I'll see what's possible. This is the one I'm constantly reverting after my auto format on save.

**Comment by bricelam** [comment]
Closing for now. Tracking work via #25355.

**Pull request closed without merging**